CHANGELOG.md

```diff
@@ -1,5 +1,173 @@
+    - [Documentation](#documentation)
+    - [Devops](#devops)
+
+
+### Making RabbitMQ optional
+
+The RabbitMQ message broker service is now optional for running AiiDA.
+The requirement was added in AiiDA v1.0 when the engine was completely overhauled.
+Although it significantly improved the scaling and responsiveness, it also made it more difficult to start using AiiDA.
+As of v2.6, profiles can now be configured without RabbitMQ, at the cost that the daemon can not be used and all processes have to be run locally.
+
+### Simplifying profile setup
+
+With the removal of RabbitMQ as a hard requirement, combined with storage plugins that replace PostgreSQL with the serverless SQLite that were introduced in v2.5, it is now possible to setup a profile that requires no services.
+A new command is introduced, `verdi presto`, that automatically creates a profile with sensible defaults.
+This now in principle makes it possible to run just the two following commands on any operating system:
+```
+pip install aiida-core
+verdi presto
+```
+and get a working AiiDA installation that is ready to go.
+
+### Improved test fixtures without services
+
+Up till now, running tests would always require a fully functional profile, which meant that PostgreSQL and RabbitMQ had to be available.
+As described in the section above, it is now possible to set up a profile without these services.
+This new feature is leveraged to provide a set of `pytest` fixtures that provide a test profile that can be used essentially on any system that just has AiiDA installed.
+To start writing tests, simply create a `conftest.py` and import the fixtures with:
+```python
+pytest_plugins = 'aiida.tools.pytest_fixtures'
+```
+The new fixtures include the `aiida_profile` fixture which is session-scoped and automatically loaded.
+The fixture creates a temporary test profile at the start of the test session and automatically deletes it when the session ends.
+For more information and an overview of all available fixtures, please refer to [the documentation on `pytest` fixtures](https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/plugins.html#plugin-test-fixtures).
+
+### Improvements and changes to caching
+
+A number of fixes and changes to the caching mechanism were introduced (see the [changes](#changes) subsection of the [full list of changes](#full-list-of-changes) for a more detailed overview).
+As a result, a database migration had to be added in order to reset all existing hashes.
+You will be automatically prompted to migrate your database after updating your installation.
+After the migration has been completed, you can use `verdi node rehash` to recreate the hashes according to the new algorithm and repopulate the cache.
+
+Since its introdcuction, the cache would essentially be reset each time AiiDA or any of the plugin packages would be updated, since the version of these packages were included in the calculation of the node hashes.
```

Comment:
```suggestion
Since its introduction, the cache would essentially be reset each time AiiDA or any of the plugin packages would be updated, since the version of these packages were included in the calculation of the node hashes.
```